src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerIntellisenseFilter.cs

```diff
@@ -89,7 +98,12 @@ void executeNextCommandTarget()
                 // legacy stuff they interfaced with. That means we get SCROLLUP if the user
                 // types escape, so treat SCROLLUP like CANCEL. It's actually a CANCEL.
                 case VSConstants.VSStd2KCmdID.SCROLLUP:
-                    ExecuteCancel(subjectBuffer, contentType, executeNextCommandTarget);
+                    ExecuteCancel(subjectBuffer, contentType, () =>
```

Comment:
💭 It's not clear to me why only two of the four uses of `executeNextCommandTarget` changed.

Reply:
these 2 handle one command but execute another (```SCROLLUP``` -> ```Cancel``` and ```TYPECHAR``` -> ```ShowMemberList```). ```executeNextCommandTarget``` captures the original handled command so we cannot use it.

Reply:
Why did it work before?

Reply:
Previously it didn't matter because both Cancel and ShowMemberList commands were pretty much guaranteed to be handled by Roslyn command handlers inside ```ICommandHandlerService``` (poorly named ```CurrentHandlers``` property) - ```executeNextCommandTarget``` was never called.

Now we have a chain of (```ICommandHandlerService```: legacy command handlers if any) -> (```IEditorCommandHandlerService```: migrated command handlers) and all Roslyn handlers are actually in the latter (former is kept for TS/F#). So properly passing executed command matters now.